### ISSUE TITLE
Traceback for error messages

### ISSUE THREAD

**Issue by distahl** [dshafik/php7-mysql-shim#53]
Hi,

First of all. Thank you for this library. It is really helpful.

Now my question:
Would it be possible to catch the errors of the mysqli_ functions and add a little traceback before giving the error?
Reason is, I'm using your lib as "auto_prepend_file" and if some functions throws an error, it looks like this:
```
mysql_num_rows() expects parameter 1 to be resource, null given in /etc/opt/rh/rh-php70/php.d/autoload/php7-mysql-shim.php on line 740
```
Unfortunately, with this error I'm not able to find the origin of this. I only know that something was using your libs function and then this was throwing an error. But I don't know the origin of this.

**Comment by naaando** [comment]
This library saves lives, but the error tracing is really missing, I'll work on a fix for this.

**Comment by distahl** [comment]
Great. Thank you so much :-)

**Comment by dshafik** [comment]
Fixed by #54
